### ISSUE TITLE
External members in tools 

### ISSUE THREAD

**Issue by themightychris** [free-explorers/tribu#1]
External members is a great idea! I added one to my Tribu but now I'm not seeing them showing up in tools like expenses or lists. It seems like they should and would be extremely useful

**Comment by themightychris** [comment]
nevermind, I just hadn't marked their presence as confirmed

maybe it would be good UX-wise to always show all group members, but sort unconfirmed ones to the bottom with some styling to show they're unconfirmed. Further, maybe being confirmed or not is just informative and doesn't drive whether you can plan against them in tools

**Comment by PapyElGringo** [comment]
Yeah I questionned myself about the exact same issue you had and ended up to the same conclusion as you.
